core/sessions/reaper.go

```diff
@@ -13,6 +13,10 @@ type sessionReaper struct {
 	db     *sql.DB
 	config SessionReaperConfig
 	lggr   logger.Logger
+
+	// Listen to this for testing via sr.SignalCh()
+	// to be notified when the reaper has run.
+	signalCh chan struct{}
```

Comment:
Would it make sense to name it a bit more semantically? e.g. `runSignal`?
Also, we often close channels to signal/notify, so it might help to clarify something about how a signal is sent after each reaper run.
```suggestion
	// Receive from this for testing via sr.SignalCh()
	// to be notified after each reaper run.
	runSignal chan struct{}
```
:shrug: